index.html

```diff
@@ -299,6 +299,10 @@ <h3>
               </td>
             </tr>
           </table>
+          <p>
+Other cryptographic suites for NIST RSA, EASC DSA, or SM9 IBSA may be produced
```

Comment:
Should we leave a window open for other cryptosuites that come into use during the WG?
```suggestion
Other cryptographic suites for NIST RSA, EASC DSA, or SM9 IBSA may be produced
```